sandman/source/timer.cpp

```diff
@@ -4,81 +4,82 @@
 	#include <Windows.h>
 #endif // defined (_WIN32)
 
-#include <time.h>
+#include <ctime>
 
 // Functions
 //
 
 // Get the current time.
 //
-// p_Time:	(Output) The current time.
+// timePoint:	(Output) The current time.
 //
-void TimerGetCurrent(Time& p_Time)
+void TimerGetCurrent(Time& timePoint)

```

Comment:
It was the case that when the variables `p_Time` and `l_Time` were renamed, there was shadowing because they were both `time`.

I'll rename `timePoint` to `time`.